css/css-pseudo/reference/selection-input-011-ref.html

```diff
@@ -19,6 +19,33 @@
     }
   </style>
 
+  <script>
+  function startReference()
+  {
+  document.getElementById("ref").focus();
+  /*
+  Some browsers, like Chromium 80+ or Safari 14+, will
+  transfer focus to a selected element like a text input
+  and therefore style the border of such element according
+  to an user agent stylesheet rule, such as:
+  :focus {outline: -webkit-focus-ring-color auto 1px;} .
+  So, we deliberately trigger such focus with the focus()
+  method in the reference file.
+  */
+  document.getElementById("ref").style.caretColor = "yellow";
```

Comment:
Please add a blank line in order to group a comment and code.

Reply:
Done in 961685d